swarm/src/lib.rs

```diff
@@ -821,6 +823,7 @@ where
                 endpoint,
                 other_established_connection_ids,
                 concurrent_dial_errors,
+                established_in: time_taken,
```

Comment:
Why call this `time_taken`? Why not be consistent with `established_in` used thus far?

Reply:
The answer to the question is context and communicating intent, but I'll change it.